recipes/blockclust/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: blockclust
+  version: 1.1.0
+
+source:
+  git_url: https://github.com/pavanvidem/blockclust
+  git_rev: c4b8a985fcb615b2b4356a242ff1bc60b672f8e1
```

Comment:
it is recommended to use following url format: `https://github.com/pavanvidem/blockclust/archive/c4b8a985fcb615b2b4356a242ff1bc60b672f8e1.tar.gz` instead of git_url.

Reply:
It is of course even better if you could tag a release.